### ISSUE TITLE
Introduce Get Started page that will be open if there are no workspaces

### ISSUE THREAD

**Issue by sleshchenko** [eclipse-che/che#15520]
### Is your enhancement related to a problem? Please describe.
Currently, a user gets quite a load page `New Workspace` from the very beginning. 
![Screenshot_20191218_111508](https://user-images.githubusercontent.com/5887312/71072995-3a8fe880-2188-11ea-9cea-c47e535891e2.png)

To provide better UX it's needed to introduce Get Started page with easy to go stacks in displayed as cards.
![68426915-2b744e80-0177-11ea-8a6c-8e9e056dbc10](https://user-images.githubusercontent.com/5887312/71073331-d4f02c00-2188-11ea-8b5f-60d14ce8588a.png)
For advanced users, there will be options: `Import From Git` and `Create a Custom Workspace`.

To deliver this page earlier few simplifications will be made:
1. Instead of Languages/Technologies filter, there will be one search input that will do a full-text matching. The same we use for stacks searching.
![Screenshot_20191218_125155](https://user-images.githubusercontent.com/5887312/71080139-3d450a80-2195-11ea-8c94-08042c8f26e3.png)
A dedicated issue will be created to introduce Language/Technologies filters because seems it needs che devfile registry model changes.

2. There is no a page we can reuse for `Import From Git`, a dedicated issue will be registered to handle this workflow.
For `Create a Custom Workspace` workflow we're going to reuse existing `Create Workspace` page. If needed a dedicated issue should be registered to improve this page.
![Screenshot_20191218_111643](https://user-images.githubusercontent.com/5887312/71073246-a70ae780-2188-11ea-8302-1e2a8d009be8.png)

**Comment by sleshchenko** [comment]
It's already implemented and merged within https://github.com/eclipse/che/pull/15642

**Comment by l0rd** [comment]
@olexii4 I have a couple of comments on the PR that you have merged:
- If compared to the [original design](https://user-images.githubusercontent.com/52013984/68426915-2b744e80-0177-11ea-8a6c-8e9e056dbc10.png) it didn't have the "Create & Open" buttons: the workspace was supposed to be created as soon as the user clicks on the card
- The "Select Template" wording is not correct because those are not template: the user is not able to customize it with his git repo or any other component of the stack. I would rephrase it "Select Sample" 

cc @slemeur

**Comment by l0rd** [comment]
@olexii4 another thing that is missing and I think it's important to keep is the footer with the version of Che. That's missing in the new Getting Started page.

**Comment by olexii4** [comment]
@l0rd It is not clear for me  about  the footer.
![getStarted2](https://user-images.githubusercontent.com/6310786/72719915-79e39780-3b81-11ea-8ea0-43fa94267c28.png)
 Is existing footer for the new 'getting started' page good enough?

**Comment by l0rd** [comment]
When I have tested on Friday I could not see any footer at all. I will check again later

**Comment by olexii4** [comment]
@l0rd I have created a new issue https://github.com/eclipse/che/issues/15749

**Comment by sleshchenko** [comment]
> When I have tested on Friday I could not see any footer at all. I will check again later

@l0rd You're right. Seems it depends on the browser. I've registered an issue for it https://github.com/eclipse/che/issues/15776

**Comment by sleshchenko** [comment]
I've just discovered that product.json has the following fragment:
```
{

  "workspace": {
    "priorityStacks": ["JBoss EAP", "Java 1.8", "Vert.x", "Spring Boot", "Wildfly Swarm", "Red Hat Fuse"]
  }
```
And I wonder if we need such prioritizing on Get Started Page? @nickboldt @slemeur @l0rd WDYT?
